config/default.yml

```diff
@@ -2776,6 +2776,13 @@ Style/GuardClause:
   # needs to have to trigger this cop
   MinBodyLength: 1
 
+Style/HashEachMethods:
+  Description: 'Use Hash#each_key and Hash#each_value.'
+  StyleGuide: '#hash-each'
+  Enabled: pending
+  VersionAdded: '0.79'
```

Comment:
Should this be updated to `'0.80'`? Looks like `0.79` was released already.

Reply:
Sure.